packages/core/src/views/account-center/Index.svelte

```diff
@@ -28,8 +29,6 @@
 <style>
   .container {
     padding: 16px;
-    max-width: 364px;
-    min-width: 348px;
```

Comment:
I think that we want to set `width: 348px` here so that the account center widget is the correct width. The designs are for `316px` width, but we add `32px` for padding on either side.

Reply:
After chatting to @gesquinca we want this container width to be `100%` and then we need a dynamic style on the container based on the device type:
```javascript
<div
  class="container flex flex-column absolute"
  style={`${accountCenterPositions[settings.position]} ${device.type === 'mobile' ? 'max-width: 352px;' : ''}`}
>
  {#if !settings.expanded && !settings.minimal}
    <!-- minimized -->
    <Minimized />
  {:else if !settings.expanded && settings.minimal}
    <!-- micro -->
    <Micro />
  {:else}
    <!-- maximized -->
    <Maximized />
  {/if}
</div>
```
The goal is that on mobile devices that the UI will cover the full width of the device (minus padding on either side) and then on anything other than mobile devices it will have a width of 352px (320px width plus 16px of padding on either side).

Reply:
Actually, going for a more dynamic approach we can use media queries instead:

```css
.container {
    padding: 16px;
    font-family: var(--onboard-font-family-normal, var(--font-family-normal));
    width: 100%;
  }
  
  @media all and (min-width: 428px) {
    .container {
      max-width: 352px;
    }
  }
```